recipes/gwframes/meta.yaml

```
@@ -0,0 +1,62 @@
+{% set name = "GWFrames" %}
+{% set version = "1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # we have to use git_url here to get the submodules needed for the
+  # build; see <https://github.com/conda-forge/xgboost-feedstock/issues/22>
+  git_url: https://github.com/moble/{{ name }}.git
+  git_rev: 792a341f77fa838bcdfd78940850205bd60c5ba4
```

Can you add a tag to the repository so this can be:
```suggestion
  git_rev: {{ version }}
```